### ISSUE TITLE
Awaiting deno_cache upstream fixes

### ISSUE THREAD

**Issue by jollytoad** [jollytoad/deno_import_content#1]
Until [AuthTokens is fixed in deno_cache](https://github.com/denoland/deno_cache/issues/17), the following import can be used (if you require DENO_AUTH_TOKENS support):

```ts
import { importText } from "https://raw.githubusercontent.com/jollytoad/deno_import_content/with-AuthTokens-fix/mod.ts";
```

**Comment by jollytoad** [comment]
I've released 0.0.4 that depends directly on a fork of `deno_cache` that contains fixes for the AuthTokens issue and also for the [write permissions](https://github.com/denoland/deno_cache/issues/20) issue.

**Comment by jollytoad** [comment]
Fixed with v1.0.0 release
